lib/options.js

```diff
@@ -114,24 +129,45 @@ module.exports = {
         default: false,
         description: 'MISSING_IN_SCHEMA indicates that an extra parameter was included in the request. For most ' +
           'use cases, this need not be considered an error.',
-        external: true
+        external: true,
+        usage: ['VALIDATION']
       },
       {
         name: 'Show detailed body validation messages',
         id: 'detailedBlobValidation',
         type: 'boolean',
         default: false,
         description: 'Determines whether to show detailed mismatch information for application/json content ' +
-          'in the request/response body',
-        external: true
+          'in the request/response body.',
+        external: true,
+        usage: ['VALIDATION']
       }
     ];
 
+    // Filter options based on criteria
+    if (_.isObject(criteria)) {
```

Comment:
I think we can directly pass the criteria object to _.filter, like `_.filter(optsArray, criteria)`

Reply:
Will not work in case of criteria being an array.